core/src/components/item/item.scss

```diff
@@ -521,35 +515,6 @@ button, a {
   --highlight-background: var(--highlight-color-invalid);
 }
 
-:host(.item-interactive.ion-invalid) ::slotted([slot="helper"]) {
-  display: none;
-}
-
-::slotted([slot="error"]) {
-  display: none;
-
-  color: var(--highlight-color-invalid);
-}
-
-:host(.item-interactive.ion-invalid) ::slotted([slot="error"]) {
-  display: block;
-}
-
-// Item Datetime
-// --------------------------------------------------
-
-:host(:not(.item-label)) ::slotted(ion-datetime) {
-  --padding-start: 0;
-}
```

Comment:
Why was this style removed? It doesn't look related to any of the deprecated APIs.

Reply:
This is part of the form control + item logic. As part of this refactor we are removing the tight coupling between items and form controls.